server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -189,6 +193,8 @@ public CompletableFuture<PushResultDto> push(
             CommitMessageDto commitMessage,
             @RequestConverter(ChangesRequestConverter.class) Iterable<Change<?>> changes) {
         checkPush(repository.name(), changes);
+        meterRegistry.counter("commits.push", "project", repository.parent().name(), "repository", repository.name())
```

Comment:
Optional) I agree it's probably useful to have detailed HTTP-level metrics tailored to CentralDogma.
In interest of generalizing in the future what do you think of adding tags corresponding to the service class and method name instead?

e.g.
```suggestion
        meterRegistry.counter("centraldogma.server.requests", "project", repository.parent().name(), "repository", repository.name(), "service": "ContentServiceV1", "method": "push")
```

Reply:
It is a nice idea. `project` and `repository` tags are added to all `api` metrics by default. If not exists, `none' is set instead.